app/git/models.py

```diff
@@ -58,16 +62,60 @@ def __str__(self):
         """Return the string representation of a model."""
         return f"[{self.category}] {self.handle}"
 
+    def update_data(self, data):
+        """Update the data field if it has changed."""
+        if self.data != data:
+            self.data = data
+            self.save()
+
+    @classmethod
+    def create_user_cache(self, user):
+        """Create a user cache object"""
+        return GitCache(handle=user, category=GitCache.Category.REPO)
+
     @classmethod
     def get_user(self, handle):
         """Utility function to retreive a user object"""
         try:
-            return self.objects.get(category=GitCache.Category.USER, handle=handle)
+            return self.objects.get(handle=handle, category=GitCache.Category.USER)
         except self.DoesNotExist:
             raise
 
-    def update_data(self, data):
-        """Update the data field if it has changed."""
-        if self.data != data:
-            self.data = data
-            self.save()
+    @classmethod
+    def create_repo_cache(self, user, repo):
+        """Create a repository cache object"""
+        return GitCache(handle=f"{user}/{repo}", category=GitCache.Category.REPO)
```

Comment:
Can you change the `self` variable to `cls` for all the methods using with the `@classmethod` decorator (https://pep8.org/#function-and-method-arguments)

Reply:
@ksolo Done, I have pushed these changes.